package.json

```diff
@@ -32,12 +32,22 @@
   ],
   "author": "",
   "license": "MIT",
-  "devDependencies": {
-    "assemble-contrib-i18n": "0.1.*",
+  "dependencies": {
     "bootstrap-sass": "3.3.1",
     "code-prettify": "^0.1.0",
     "datatables": "1.10.13",
     "es5-shim": "2.3.0",
+    "html5shiv": "^3.7.3",
+    "jquery": "2.1.4",
+    "jquery-validation": "1.15.1",
+    "load-grunt-tasks": "^3.2.0",
```

Comment:
Is this a regular dependency because of the recursion?

Reply:
Ooops, nop, copy/past mistake